src/olympia/constants/activity.py

```diff
@@ -1119,6 +1119,50 @@ class REQUEST_LEGAL(_LOG):
     cinder_action = DECISION_ACTIONS.AMO_LEGAL_FORWARD
 
 
+class HELD_ACTION_REJECT_VERSIONS(_LOG):
+    id = 199
+    action_class = 'reject'
+    format = _('{addon} {version} rejection held for further review.')
```

Comment:
It won't be shown to developers (it's an admin event), so it doesn't need to be translated. Same for the others below (and also `REQUEST_LEGAL` above if you want to do a drive-by)